### PR TITLE
Language from default language library from gmod

### PR DESCRIPTION
**Pull request by NekerSqu4w**
Simple request for adding this library

### CODE REVIEW

**Comment by ANormalTwig** [comment]
Your indentation seems off at your return statements, make sure to use tabs instead of spaces.

**Comment by NekerSqu4w** [comment]
That what i do, but idk why it's added space like that

**Comment by UnrealEugene** [comment]
Is it possible to replace existing language phrases? Or to undo these changes?

**Comment by NekerSqu4w** [comment]
Hmmm, with default language function no, you can only add more query word but you can't modify it,
Or maybe if i do a custom language system maybe

**Comment by ANormalTwig** [comment]
If you can't undo it modify them then the permission should be defaulted to owner only.

**Comment by NekerSqu4w** [comment]
.. what the id for owner only ?

**Comment by thegrb93** [comment]
I have a feeling adding too much will crash the game. Will need to limit it. Not sure what purpose this is for anyway.

File: lua/starfall/libs_cl/language.lua [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+
+--- Adds a language item. Language placeholders preceded with "#" are replaced with full text in Garry's Mod once registered with this function.
+-- @param string placeholder Query id
+-- @param string fulltext Phrase from query
+function language_lib.add(placeholder,fulltext)
+	checkpermission(instance, placeholder, "language.add")
+	checkluatype(placeholder, TYPE_STRING)
+	checkluatype(fulltext, TYPE_STRING)
+	language.Add(placeholder,fulltext)
```
Comment (thegrb93) [inline lua/starfall/libs_cl/language.lua:25]: placeholder and fulltext length needs to be limited

**Review by thegrb93 — CHANGES_REQUESTED** [review]

**Comment by thegrb93** [comment]
Will close this stale PR if no work is done on it.

**Comment by Vurv78** [comment]
If we just want a localization library, could be done much better. This is by the same guy that made middleclass, too https://github.com/kikito/i18n.lua

**Comment by thegrb93** [comment]
I think they more wanted it in order to manipulate engine strings, but closing due to inactivity.

**Pull request closed without merging**